build.gradle

```diff
@@ -50,40 +54,46 @@ configurations {
 
 dependencies {
 	enigmaRuntime "net.fabricmc:stitch:0.1.2.49+"
-	enigmaRuntime "cuchaz:enigma:0.13.1.99:all"
+	enigmaRuntime "cuchaz:enigma:0.13.1.114+:all" // Update version above as well!
```

Comment:
is 114+ correct? should it be `0.13.1.+`?

Reply:
Still locked on enigma. @modmuss50 Can you do us a favor by reverting https://github.com/FabricMC/Enigma/commit/44dd51db443839788fe23c55d6036d8fa3ba692b ?